src/Node.js

```diff
@@ -4,16 +4,43 @@ import CollapsibleItem from './CollapsibleItem';
 import { Property, Properties } from './properties';
 
 class Node extends Component {
+  constructor(props) {
+    super(props);
+
+    this.onMouseMove = this.onMouseMove.bind(this);
+    this.onToggle = this.onToggle.bind(this);
+  }
+
+  onMouseMove(e) {
```

Comment:
This file is a bit confusing with the `onMouseMove` and `onToggle` properties and methods. It seems it will be very easy to mix both when we edit this in the future.

I wonder if this could be made more clean with more different names, such as `handleMouseMove` and `handleToggle` for these methods.

Reply:
sorry, I don't really get what you mean. Do you suggest to rename all event handlers from `onSomething` -> `handleSomething`?

Reply:
I mean that these two blocks in `Node.render` are too similar

```
      <CollapsibleItem
        ...
        toggle={() => onToggle(id)}
        onMouseMove={this.onMouseMove}
```

```
        <Children
        ...
          onToggle={onToggle}
          onMouseMove={onMouseMove}
        />
```

If someone new comes to this project and has to modify something here it will be very easy to mix `this.onMouseMove` with `this.props.onMouseMove`.

To add to the confusion, it looks like [`Node.onToggle`](https://github.com/smacker/uast-viewer/blob/06cb02ed6023b99d0b42b7ca10e02b0dbba4bf6c/src/Node.js#L22) is defined but not used, being replaced by `() => onToggle(id)`.

What I suggest is to have a different name for the `Node` methods, so `Node.render` would be:

```
      <CollapsibleItem
        ...
        toggle={this.handleToggle}
        onMouseMove={this.handleMouseMove}
```

```
        <Children
        ...
          onToggle={onToggle}
          onMouseMove={onMouseMove}
        />
```

Of course this is my point of view, if you strongly feel the current code is clear enough, keep it.

Reply:
thanks for the explanation, makes perfect sense!